src/main.js

```diff
@@ -6,8 +6,9 @@ import globalComponents from './plugins/globalComponents';
 import i18n from '@/plugins/vue-i18n';
 import './api/apiConfig';
 import './assets/tailwind.css';
+import ElementUI from 'element-ui'
 
-
+Vue.use(ElementUI);
```

Comment:
As the navbar dropdown will be displayed on every page (in the navigation), I have imported the ElementUI globally